staging/src/k8s.io/apiserver/pkg/endpoints/filters/audit.go

```diff
@@ -147,6 +148,7 @@ func processAuditEvent(sink audit.Sink, ev *auditinternal.Event, omitStages []au
 			return
 		}
 	}
+	ev.CreationTimestamp = metav1.NewTime(time.Now())
```

Comment:
I am not very sure.
If ev.meta == nil, will  this panic ?

Reply:
No, it won't